Fix codespell not checking spelling (#11)
RebelToolbox/RebelDocumentation: about/faq.rst

There are several reasons for this:

> - **Code maintenance and surface for bugs.** Every time we accept new code in

Reviewer: ```suggestion
- **Code maintenance and surface for bugs.** Every time we accept new code into
```
Just a thought on what sounds better to me.

>   the Rebel Engine repository, existing contributors often take the responsibility of

Reviewer: ```suggestion
  the Rebel Engine repository, existing contributors take responsibility of
```
Is it "often" or always? If it is always, then I'd remove the word "often". Same with the article "the" before responsibility.

Author: Good question. However, you highlight a bigger issue: although we want to ensure that the code is easy to maintain and bug-free, I'm not sure this is a valid reason for not accepting new code. We need to clearly describe and explain the Rebel philosophy, but that is beyond the scope of this PR.

>   maintaining it. Some contributors don't always stick around after getting

Reviewer: ```suggestion
  maintaining it. Some contributors step back and focus on other projects after getting
```
IMHO "don't always stick around" is not a professional way of speaking about fellow contributors.
